integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/iud/TestShowIUDRowCount.scala

```diff
@@ -0,0 +1,60 @@
+package org.apache.carbondata.spark.testsuite.iud
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.test.util.QueryTest
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+
+class TestShowIUDRowCount extends QueryTest with BeforeAndAfterEach with BeforeAndAfterAll {
+
+  override protected def beforeAll(): Unit = {
+    dropTable("iud_rows")
+  }
+
+  override protected def beforeEach(): Unit = {
+    dropTable("iud_rows")
+  }
+
+  override protected def afterEach(): Unit = {
+    dropTable("iud_rows")
+  }
+
+  test("Test show load row count") {
+    sql("""create table iud_rows (c1 string,c2 int,c3 string,c5 string)
+        |STORED BY 'org.apache.carbondata.format'""".stripMargin)
+    checkAnswer(
+      sql(s"""LOAD DATA LOCAL INPATH '$resourcesPath/IUD/dest.csv' INTO table iud_rows"""),
```

Comment:
why  `sql()` function in QueryTest get different plan to self made spark context or beeline